libsolidity/formal/CHC.h

```diff
@@ -181,19 +181,14 @@ class CHC: public SMTEncoder
 	/// @returns <false, model> otherwise.
 	std::pair<smtutil::CheckResult, smtutil::CHCSolverInterface::CexGraph> query(smtutil::Expression const& _query, langutil::SourceLocation const& _location);
 
-	void addVerificationTarget(ASTNode const* _scope, VerificationTarget::Type _type, smtutil::Expression _from, smtutil::Expression _constraints, smtutil::Expression _errorId);
-	void addVerificationTarget(ASTNode const* _scope, VerificationTarget::Type _type, smtutil::Expression _errorId);
-	void addVerificationTarget(frontend::Expression const& _scope, VerificationTarget::Type _type, smtutil::Expression const& _target);
-	void addAssertVerificationTarget(ASTNode const* _scope, smtutil::Expression _from, smtutil::Expression _constraints, smtutil::Expression _errorId);
+	void errorConditionEncountered(ASTNode const * const _errorNode, VerificationTarget::Type _type, smtutil::Expression const& _errorCondition);
```

Comment:
Or actually, `addVerificationTarget` wouldn't be too bad, or `verificationTargetEncountered`, since it uses `CHCVerificationTarget` to fill `m_verification[Pre]Targets`.

Reply:
Used the last suggestion.